avro/src/main/java/com/fasterxml/jackson/dataformat/avro/AvroParser.java

```diff
@@ -255,6 +264,16 @@ public JsonLocation getCurrentLocation()
     @Override
     public abstract JsonToken nextToken() throws IOException;
 
+    @Override
+    public boolean canReadTypeId() {
```

Comment:
This method is typically expected to base decision just on format's capability to encode type information...